pkgs/applications/graphics/krita/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, cmake, kdelibs, attica, perl, zlib, libpng, boost, mesa
+, kdepimlibs, createresources ? null, eigen, qca2, exiv2, soprano, marble, lcms2
+, fontconfig, freetype, sqlite, icu, libwpd, libwpg, pkgconfig, poppler_qt4
+, libkdcraw, libxslt, fftw, glew, gsl, shared_desktop_ontologies, okular
+, libvisio, kactivities, mysql, postgresql, freetds, xbase, openexr, ilmbase
+, libodfgen, opencolorio, openjpeg, pstoedit, librevenge, oxygen_icons
+ }:
+
+stdenv.mkDerivation rec {
+  name = "krita-2.9.5";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/calligra-2.9.6/calligra-2.9.6.tar.xz";
+    sha256 = "1151k0x2s2k9p833l5hhgi8ph55k5b76dpw5a034abry2cvl2mkq";
+  };
+
+  nativeBuildInputs = [ cmake perl pkgconfig ];
+
+# TODO: package Vc, libWPS, Spnav, m2mml, LibEtonyek, poppler-qt4-xpdf-headers
+# not found: xbase, openjpeg(too new)
+
+  buildInputs = [ kdelibs attica zlib libpng boost mesa kdepimlibs
+    createresources eigen qca2 exiv2 soprano marble lcms2 fontconfig freetype
+    sqlite icu libwpd libwpg poppler_qt4 libkdcraw libxslt fftw glew gsl
+    shared_desktop_ontologies okular libodfgen opencolorio openjpeg
+    libvisio kactivities mysql.lib postgresql freetds xbase openexr pstoedit
+    librevenge oxygen_icons
+  ];
+
+  cmakeFlags = ''
```

Comment:
It would be better to set this as a list:
```
cmakeFlags = [
  "-DPRODUCTSET=KRITA"
];
```
It will work when there is only one flag, but I'm not sure it will work if we need to add more flags later.